sdk/eventhubs/azure-messaging-eventhubs/src/main/java/com/azure/messaging/eventhubs/EventHubClientBuilder.java

```diff
@@ -85,6 +82,16 @@
      * The name of the default consumer group in the Event Hubs service.
      */
     public static final String DEFAULT_CONSUMER_GROUP_NAME = "$Default";
+    /**
+     * The minimum value allowed for the prefetch count of the consumer.
+     */
+    static final int MINIMUM_PREFETCH_COUNT = 1;
+    /**
+     * The maximum value allowed for the prefetch count of the consumer.
+     */
+    static final int MAXIMUM_PREFETCH_COUNT = 8000;
```

Comment:
Is this max defined by the service?

Reply:
This is something that existed in Track 1. And they updated this value manually in the code. I ported it as-is.